operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md

```diff
@@ -3,24 +3,40 @@
 See [Configuration Management Use Cases](Configuration_Management.md#use-cases) for more information about image customization and when it should be used.
 
-Ideally image customization playbooks should be separate from node personalization playbooks. This reduces the likelihood that Ansible content will run in both modes, and reduces the need for conditional checks in the playbooks.
+## Using the `cfs_image` host group
 
-For situations when both image customization and node personalization need to be part of the same playbook, CFS provides the `cray_cfs_image` variable to distinguish between
-node personalization and image customization. When this variable is set to `true`, it indicates that the CFS session is an image customization and the playbook is targeting an image.
+During image customization, CFS will automatically add all image customization hosts into a special `cfs_image` host group in Ansible inventory.
+Plays intended for image customization can then target this group in addition to any other provided host groups.
 
-## Using `cray_cfs_image`
+To target only image customization, plays should use the following syntax.  In this example the play is targeting only images for `Compute` nodes. `&` takes the intersection of the `Compute` and `cfs_image` groups.
 
-`cray_cfs_image` can be set with [Ansible playbook conditionals](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html)
+```yaml
+hosts: Compute:&cfs_image
+```
+
+To target only node personalization, plays should use the following syntax.  In this example the play is targeting only running `Compute` nodes. `!` negates the `cfs_image` group, so that only Compute nodes that are not an image are targeted.
+
+```yaml
+hosts: Compute:!cfs_image
+```
+
+## Using the `cray_cfs_image` variable
+
+`** NOTE **` This option is no longer recommended and should only be used in small playbooks and one-off cases
+ as it is more efficient for Ansible to determine this at the host level rather than checking the `cray_cfs_image` variable for multiple tasks.
+
+CFS also provides the `cray_cfs_image` variable to distinguish between node personalization and image customization. 
+`cray_cfs_image` can be used with [Ansible playbook conditionals](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html)
 to selectively run individual tasks with `when: cray_cfs_image`, or to ignore individual tasks with `when: not cray_cfs_image`.
 
-Conditionals can also be applied to entire roles if desired \(see the external [apply Ansible conditionals to roles](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#applying-when-to-roles-imports-and-includes)\).
+Conditionals can also be applied to entire roles if desired \(see the external [apply Ansible conditionals to roles](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#conditionals-with-includes).
```

Comment:
Your text says roles so use https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#conditionals-with-roles
instead of -includes.

Unless you explicitly meant the includes anchor because it dovetails with the next sentence.

Reply:
I chose to point specifically to "includes" because that's the preferred method.  If you disagree with that those I can change it.